### ISSUE TITLE
Add a step-by-step guide on how to connect to the backend

### ISSUE THREAD

**Issue by akib22** [fossology/FOSSologyUI#181]
### Description
I'm attempting to run the project with `yarn` in development mode. However, I am unable to connect to the backend.
I followed the instructions from `readme.md`. 

node -> v16.14.0
yarn -> v1.22.17

#### Screenshots
![error screenshot](https://i.ibb.co/zsFfm9w/Screen-Shot-2022-03-14-at-11-28-00-PM.png)

**How `localhost/repo/api/v2` will work? Can anyone please explain to me?.**

**Comment by sjha2048** [comment]
Hi @akib22, do you have [FOSSology](https://github.com/fossology/fossology) Installed on your system?

**Comment by akib22** [comment]
No.

**Comment by RakeshPotnuru** [comment]
@akib22 
You have to first install [FOSSology](https://github.com/fossology/fossology/wiki/Install-from-Source) and then refer to this [wiki](https://github.com/fossology/FOSSologyUI/wiki/Installation)

**Comment by Raunakk02** [comment]
@akib22 One more thing that I would like to add is, once the [FOSSology](https://github.com/fossology/fossology/wiki/Install-from-Source) is up and running at the URL `http://localhost:8081/repo`. Then, you need to follow these two steps:

1. Setup CORS as instructed [here](https://github.com/fossology/FOSSologyUI/wiki/Installation#:~:text=Setup%20CORS%20in%20FOSSology)  

2. In the .env file of the [FOSSologyUI](https://github.com/fossology/FOSSologyUI) project, assign the value of REACT_APP_SERVER_URL as:

```
REACT_APP_SERVER_URL=localhost:8081/repo/api/v1
```

**Comment by GMishx** [comment]
Does it make sense to convert this issue to a GitHub discussion similar to https://github.com/fossology/fossology/discussions/1942 ?

**Comment by d-e-v-esh** [comment]
@GMishx Yes, I think so, I tried everything but it didn't work for me. Maybe someone else can say what is the problem.

**Comment by GMishx** [comment]
There is a fix proposed at https://github.com/fossology/fossology/pull/2195 for CORS.
